ert_shared/main.py

```diff
@@ -441,6 +441,15 @@ def start_ert_server(mode: str):
         yield
 
 
+def log_config(args: ArgumentParser, logger: logging.Logger) -> None:
+    if args.config is not None and os.path.isfile(args.config):
+        with open(args.config, "r", encoding="utf-8") as file_obj:
+            content = file_obj.read()
+            logger.debug(
+                f"Content of the configuration file ({args.config}):\n" + content
```

Comment:
Pylint will complain about non-lazy formatting here. Use
```suggestion
                "Content of the configuration file (%s):\n%s", args.config, content
```

Reply:
I would say let it complain, or even disable the complaining from pylint. The level is `DEBUG` when running `ert` anyway, so there is very little performance gain by doing lazy-formatting.

Reply:
I agree with that, I never liked that preference for old style formatting in logging. I know it is more efficient, but I would say that is a wart in the language or the library.

We would need a common policy for that, to be enforced by `pylint`, which we currently not do for this part of the code. Hence, I leave it as is for now.